rites/roman1969/src/locales/fr.ts

```diff
@@ -334,18 +347,22 @@ export const locale: Locale = {
     frederic_ozanam_founder: 'Bienheureux Frédéric Ozanam, fondateur de la Société de Saint-Vincent-de-Paul († 1853)',
     friday_of_the_passion_of_the_lord: 'Vendredi Saint',
     fridolin_of_sackingen_monk: 'Saint Fridolin, moine missionnaire († 540)',
+    fromundus_of_coutances_bishop: 'Saint Fromond, évêque de Coutances († v. 691)', // src: mr_fr_1982_ed2_coutances
     gall_of_switzerland_abbot: 'Saint Gall, abbé et missionnaire († 641 ou 646)',
+    gaud_of_evreux_bishop: 'Saint Gaug, évêque († v. 491)', // src: mr_fr_1982_ed2_coutances
```

Comment:
![Screenshot_20241012_004756_Firefox Nightly](https://github.com/user-attachments/assets/823ad3b0-6f2a-45ca-9f5a-f4cf2d72d85a)

```suggestion
    gaud_of_evreux_bishop: 'Saint Gaud, évêque († v. 491)', // src: mr_fr_1982_ed2_coutances
```

Reply:
Ah! Thanks, I thought I had fixed everything here 😄